pkg/fqdn/dnsproxy/udp.go

```diff
@@ -102,6 +102,9 @@ func listenConfig(mark int, ipv4, ipv6 bool) *net.ListenConfig {
 				if opErr == nil {
 					opErr = syscall.SetsockoptInt(int(fd), syscall.SOL_SOCKET, syscall.SO_REUSEADDR, 1)
 				}
+				if opErr == nil {
+					opErr = unix.SetsockoptInt(int(fd), unix.SOL_SOCKET, unix.SO_REUSEPORT, 1)
```

Comment:
:100: 

Unrelated to this PR, just a note that we might need to change syscall to unix in above line later.

Reply:
oh my bad, just realised that this PR is for 1.7 branch, on master branch syscall is already replaced by unix :+1: